client/js/app/validations/ExplorerValidations.js

```diff
@@ -46,6 +46,20 @@ module.exports = {
 
   },
 
+  percentile_value: {
+    
+    msg: 'Choose a Percentile Value.',
+
+    shouldRun: function(model) {
+      return model.query.analysis_type === 'percentile';
+    },
+    
+    validate: function(model) {
+      return typeof model.query.percentile === 'number';
```

Comment:
Given the name of the error message, I think the validation check here should be if it exists or not, rather than whether it's a `Number` type. Thoughts?